### PR TITLE
parsec: update to 150 [ci-skip]

### PR DESCRIPTION
**Pull request by silvernode**

### CODE REVIEW

File: srcpkgs/parsec/template [hunk 1]
```diff
@@ -21,6 +21,6 @@ do_extract() {
 do_install() {
 	tar xf "data.tar.xz" -C ${DESTDIR}
```
Comment (pullmoll) [inline srcpkgs/parsec/template]: Either use `bsdtar` or add `tar` to `hostmakedepends`.

**Comment by silvernode** [comment]
I went with bsdtar so things should work this time around.

**Comment by silvernode** [comment]
I guess it wants me to find the ID for this license, which there doesn't seem to be anything relevant on SPDX so I'll have to look into this and figure out what I need to supply to the string. Seems like I can just write"custom: proprietary" but I'm not sure.

**Comment by silvernode** [comment]
Parsec does not ship a license file with their debian package and only have a terms of service page with a license section which is one paragraph. Should I just not include the license variable at all? the Manual says if there is no license file to default to BSD-2-clause, but xlint complains that there is still no license file. Not sure what to do next. This template was previously merged with license var the way it is.

**Comment by Chocimier** [comment]
Manual says to use BSD for package with no files at all, not about lacking license. Set license="custom:proprietary" and ignore warning to use vlicense.

**Comment by silvernode** [comment]
> Manual says to use BSD for package with no files at all, not about lacking license. Set license="custom:proprietary" and ignore warning to use vlicense.

Ah, now I feel dumb, I should have followed my instinct.

**Comment by silvernode** [comment]
There, now travis wont try to build it and fail at the license. I apparently had the [ci skip] syntax wrong the first time.

**Comment by abenson** [comment]
Superceded by 92f86cffc7f0eab28634c27cc1535f0baebda842

**Pull request closed without merging**